gnomad/utils/constraint.py

```diff
@@ -810,12 +814,19 @@ def get_constraint_grouping_expr(
         groupings["transcript"] = vep_annotation_expr.transcript_id
     if include_canonical_group:
         groupings["canonical"] = hl.or_else(vep_annotation_expr.canonical == 1, False)
+    elif include_mane_select_group:
```

Comment:
I think this should just be if
```suggestion
    if include_mane_select_group:
```